docs/source/terms_and_model.rst

```diff
@@ -387,18 +387,18 @@ as an :ref:`Allele` with a :ref:`RepeatedSequenceExpression`.
 
 **Examples**
 
-Two, three, or four total copies of BRCA1:
```

Comment:
I believe we intended to use `BRCA1` and not `APOE` in this example, just so that it would be more easily distinguished from the prior example which is an `APOE` Haplotype. Let's leave this as `BRCA1`